go.mod

```diff
@@ -5,5 +5,6 @@ go 1.13
 require (
 	github.com/golang/protobuf v1.3.3
 	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.5.1
```

Comment:
Hmm... do we want to make our API depend on testify ?  I'm not saying we don't... but its worth thinking about :)

Reply:
If we have some public functions in API then I think it can be covered by unit tests. For example if we found a bug related to API functions then we can add the unit test for the bug into API repo.

Reply:
finally, I've removed the tests. I think we can cover this code in the next repository.